sysconfigs/daint/packages.yaml

```diff
@@ -162,10 +162,6 @@ packages:
     externals:
     - spec: libtool@2.4.6
       prefix: /usr
-  libxml2:
```

Comment:
Why was this removed?

Reply:
There was a version incompatability when spack picked libxml2 from /usr.